src/components/Notifications/Notifications.container.ts

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors'
+import { isConnecting } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { RootState } from '../../modules/reducer'
+import { GET_SUBSCRIPTIONS_REQUEST, getSubscriptionsRequest } from '../../modules/subscription/actions'
+import { getLoading } from '../../modules/subscription/selectors'
+import Notifications from './Notifications'
+import { MapDispatch, MapDispatchProps, MapStateProps, OwnProps } from './Notifications.types'
+
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
+  return {
+    isLoading: ownProps.isLoading || isConnecting(state) || isLoadingType(getLoading(state), GET_SUBSCRIPTIONS_REQUEST)
```

Comment:
@braianj - in which cases would you pass the `isLoading` param from an external component? 

wdyt about having all the related loading states from the store here? that should cover all the cases?

Reply:
good catch